Makefile

```diff
@@ -323,7 +321,7 @@ package-darwin: require-version require-appdmg require-svgexport darwin
 
 binaries: docker genassets linux windows darwin
 
-packages: require-version require-secrets clean binaries package-windows package-linux package-darwin
+packages: require-version require-secrets clean genconfig binaries package-windows package-linux package-darwin
```

Comment:
The packages target is a required step for release-qa, and it's already slow so it makes sense to use genconfig here.